imapclient/acl_test.go

```diff
@@ -0,0 +1,100 @@
+package imapclient_test
+
+import (
+	"testing"
+
+	"github.com/emersion/go-imap/v2"
+)
+
+// TestACL runs tests on SetACL and MyRights commands (for now).
+func TestACL(t *testing.T) {
+	client, server := newClientServerPair(t, imap.ConnStateAuthenticated)
+
+	defer client.Close()
+	defer server.Close()
+
+	client.Create("MyFolder", nil)
+	client.Create("MyFolder/Child", nil)
```

Comment:
We should handle errors here, or else this could result in some annoying-to-debug situations in tests.